common/pulp/common/constants.py

```diff
@@ -0,0 +1,17 @@
+# Call States
+
+CALL_WAITING_STATE = 'waiting'
+CALL_SKIPPED_STATE = 'skipped'
+CALL_ACCEPTED_STATE = 'accepted'
+CALL_RUNNING_STATE = 'running'
+CALL_SUSPENDED_STATE = 'suspended'
+CALL_FINISHED_STATE = 'finished'
+CALL_ERROR_STATE = 'error'
+CALL_CANCELED_STATE = 'canceled'
+
+CALL_INCOMPLETE_STATES = (CALL_WAITING_STATE, CALL_ACCEPTED_STATE, CALL_RUNNING_STATE,
+                          CALL_SUSPENDED_STATE)
+CALL_COMPLETE_STATES = (CALL_SKIPPED_STATE, CALL_FINISHED_STATE, CALL_ERROR_STATE,
+                        CALL_CANCELED_STATE)
+CALL_STATES = (CALL_WAITING_STATE, CALL_SKIPPED_STATE, CALL_ACCEPTED_STATE, CALL_RUNNING_STATE,
+               CALL_SUSPENDED_STATE, CALL_FINISHED_STATE, CALL_ERROR_STATE, CALL_CANCELED_STATE)
```

Comment:
I've updated these, and also switched them to use [SimpleNamespace](https://docs.python.org/3.3/library/types.html#types.SimpleNamespace).